README.rst

```diff
@@ -1,21 +1,31 @@
-.. image:: https://img.shields.io/pypi/v/pytest-flake8.svg
+.. image:: https://img.shields.io/pypi/v/pytest-flake8.svg?label=pytest-flake8
```

Comment:
**Original view  pytest-flake8 1.2.2:**
![image](https://github.com/user-attachments/assets/2a1d9e25-7188-4d9c-9ec5-5798471b25f7)


**Skeleton view master:**
![image](https://github.com/user-attachments/assets/0a1bb99d-a12e-4ad0-853e-1dd759640c34)


**From current PR view:**
![image](https://github.com/user-attachments/assets/55d74e8a-c097-4317-be5c-697545f715c8)

I have made some changes because, as you may notice in the documentation for version 1.2.2, there is a blue underline (very tiny) appearing between the labels tests and Ruff. This issue is likely related to the SVG extension at the end. In the skeleton, it may be unnoticeable due to other styles being present so to enhance clarity and coherence for this project, I have added a label and made adjustments to ensure that the pages appear more consistent and user-friendly.

I haven't looked into the skeleton project. I made the changes with varying degrees of familiarity with HTML and by reviewing the badges. I would prefer to focus on delivering a functioning version of pytest-flake8, and I would like others to benefit from this as well. I also apologize in advance for my English, as it is not at the highest level, and I often hear from people that it sounds very harsh, which is not my intention.

Reply:
Very strange situation, and it looks good on PyPI...

![image](https://github.com/user-attachments/assets/b0970582-241f-4825-a370-14fa2cce07d0)

Reply:
@jaraco Do you have any suggestion in which direction we should go with this delivery?

Reply:
I recognize the skeleton project isn't something that's familiar to everyone. It's a way of dealing with the common concerns of project management across projects.

The skeleton project itself is never released, so whatever rendering you see of the badges in skeleton is ignorable. What really matters is what is rendered for the downstream projects (after substituting real project names for concrete projects).

I'm aware there are some issues with the rendering of badges in some situations. I'm unwilling to "fix" the rendering of the badges for individual projects. If the issue can be fixed for all projects, I'm good with that, but it needs to happen in the skeleton. My recommendation - just revert all changes to the badges and instead contribute the _same_ changes upstream in the skeleton. The only thing I'm likely to accept in this project is an additional badge that appears after the "skeleton" badge.

Reply:
In other words, let's just revert these changes and deal with them separately, so they aren't blocking the other changes.

Reply:
Thank you.